src/requirements/data/constants.ts

```diff
@@ -1,12 +1,18 @@
-/**
- * Course id for special course that satisfies total academic credits requirement.
- */
-export const CREDITS_COURSE_ID = 10;
-
-/**
- * Course id for special course that satisfies FWS requirement.
- */
-export const FWS_COURSE_ID = 11;
+/** Course ID for special course that satisfies no requirements. */
+export const NO_EQUIVALENT_COURSES_COURSE_ID = 10;
```

Comment:
Just curious: what's an example of a special course that satisfies no req?

Reply:
If there are no equivalent courses for an AP/IB exam, but the user wants to use the exam to override, there needs to be a dummy course + uniqueId. so, if an AP/IB exam has 0 equivalent courses, this dummy course is pushed.